.travis.yml

```diff
@@ -210,6 +210,11 @@ script:
   - date > time.txt && curl -v --fail -T time.txt ftp://localhost:8021 --user $GALAXY_USER:$GALAXY_USER_PASSWD || true
   # Test FTP Server get
   - curl -v --fail ftp://localhost:8021 --user $GALAXY_USER:$GALAXY_USER_PASSWD
+  
+  # Test CVMFS
+  - docker_exec_run bash -c "service autofs start"
```

Comment:
Not sure about that, but it gets an error when trying to run on the base image (no compose). It's still the privilege error that tells me that docker probably isn't running with --privileged (or perhaps autofs module is not in the host kernel).